packages/video_player/video_player_avfoundation/darwin/video_player_avfoundation/Sources/video_player_avfoundation/FVPVideoPlayerPlugin.m

```diff
@@ -587,6 +613,13 @@ - (FlutterError *_Nullable)onListenWithArguments:(id _Nullable)arguments
   // This line ensures the 'initialized' event is sent when the event
   // 'AVPlayerItemStatusReadyToPlay' fires before _eventSink is set (this function
   // onListenWithArguments is called)
+  // and also send error in similar case with 'AVPlayerItemStatusFailed'
```

Comment:
by "similar case", do you mean AVPlayerItemStatusReadyToPlay fires before _eventSink is set? Can you be more explicit here in the comment? (Sorry I have some trouble understanding this bug fix)

Reply:
Yes I was following up with that comment above. But this time with `AVPlayerItemStatusFailed` instead of `AVPlayerItemStatusReadyToPlay`. If anything fires before is event sink set then it could not be sent. Would that comment be better if I added "in similar case as above"?